.github/workflows/integration.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     name: ${{ matrix.os }} JDK 8
     strategy:
       matrix:
-        os: [ubuntu-18.04, ubuntu-20.04]
+        os: [ubuntu-18.04, ubuntu-20.04, ubuntu-22.04]
```

Comment:
I'm not sure if we should adopt `ubuntu-22.04` since it's still in beta according to the [actions/virtual-environments page](https://github.com/actions/virtual-environments#available-environments).

Reply:
Yup, it is in beta currently. I added this such that we can run on this version, but I will not be marking the run as a required check (i.e. it is allowed to fail but will not block the PR from being merged).